main.py

```diff
@@ -530,12 +534,16 @@ def handle_new_switch(self, event):
             self.notify_switch_enabled(switch.id)
 
     @listen_to('.*.connection.lost')
-    def handle_connection_lost(self, event):
+    def on_connection_lost(self, event):
```

Comment:
@italovalcy @ajoaoff @rmotitsuki maybe we can try to define a naming pattern convention for `listen_to` decorated methods, like they all could be encouraged to start with `on_*` and then whenever you see a `on_*` you know that's a  `listen_to` decorated method, let me know if you'd agree or if you have other suggestion about this. having a pattern would also facilitate the code navigation, readability and easy to remember when writing tests without having to keep checking the definition all the time (to figure out if it's a `listen_to` threaded or not).

Reply:
Great suggestion, @viniarck! I agree with this suggestion. Maybe we can document this at [Best practices for Kytos NApp development](https://github.com/kytos-ng/kytos-ng.github.io/blob/master/napps/best_practices.md)

Reply:
Great idea. Let's add this on our radar. I've mapped an issue in that repo for this https://github.com/kytos-ng/kytos-ng.github.io/issues/12